README.md

```diff
@@ -221,6 +224,7 @@ PRs or via private communication:
 * Pedro Ojeda ([@pojeda](https://github.com/pojeda))
 * Samuel Omlin ([@omlins](https://github.com/omlins))
 * Dinindu Senanayake ([@DininduSenanayake](https://github.com/DininduSenanayake))
+* Alexander Bills ([@abillscmu](https://github.com/abillscmu))
```

Comment:
Would you mind putting your name alphabetically in the list as well?

Reply:
I have now put them all in alphabetical order